src/inventory.py

```diff
@@ -14,7 +14,8 @@ class Inventory():
     to manage the inventory.
     """
     def __init__(self):
-        self._moduleName = BMConfigParser().safeGet("inventory", "storage")
+        self.config = BMConfigParser()
+        self._moduleName = self.config.safeGet("inventory", "storage")
```

Comment:
shouldn't be necessary. If it causes problems anyway, it needs to be investigated in more detail